tests/sys/psa_crypto_cipher/Makefile

```
@@ -6,8 +6,9 @@ USEMODULE += ztimer_usec
 USEMODULE += psa_crypto
 
 USEMODULE += psa_cipher
+USEMODULE += psa_cipher_chacha_20
 USEMODULE += psa_cipher_aes_128_cbc
 
-CFLAGS += -DCONFIG_PSA_SINGLE_KEY_COUNT=1
+CFLAGS += -DCONFIG_PSA_SINGLE_KEY_COUNT=2
```

```suggestion
CFLAGS += -DCONFIG_PSA_SINGLE_KEY_COUNT=1
```

see my earlier comment on how to properly free the key slot.